src/sentry/static/sentry/app/views/releasesV2/list/index.tsx

```diff
@@ -71,6 +72,14 @@ class ReleasesList extends AsyncView<Props, State> {
     return [['releases', `/organizations/${organization.slug}/releases/`, {query}]];
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    super.componentDidUpdate(prevProps, prevState);
+
+    if (prevState.releases !== this.state.releases) {
+      forceCheck();
```

Comment:
What does this do? Can we add some comments?

Reply:
Sure, it's been added:

Manually trigger checking for elements in viewport. 
Helpful when LazyLoad components enter the viewport without resize or scroll events.
https://github.com/twobin/react-lazyload#forcecheck

Reply:
Actually, I updated it to be more descriptive why we are doing it in the first place.

/**
       * Manually trigger checking for elements in viewport.
       * Helpful when LazyLoad components enter the viewport without resize or scroll events,
       * https://github.com/twobin/react-lazyload#forcecheck
       *
       * HealthStatsCharts are being rendered only when they are scrolled into viewport.
       * This is how we re-check them without scrolling once releases change as this view
       * uses shouldReload=true and there is no reloading happening.
       */

Reply:
Much more helpful, thanks!